kernel-module/picoevb-rdma.c

```diff
@@ -166,6 +166,7 @@ static void pevb_p2p_free_callback(void *data)
 	mutex_unlock(&pevb_file->lock);
 
 	nvidia_p2p_free_page_table(cusurf->page_table);
+	kfree(cusurf->page_table);
```

Comment:
This is not needed as free_page_table call above should take care of that.

Reply:
This change was required by our application on a Jetson AGX Xavier running Jetpack 4.6 to avoid a kernel memory leak that would eventually (after several weeks) cause the OS to crash. That said, I am no kernel expert nor am I sure what the internals of nvidia_p2p_free_page_table look like for a Xavier system. But I will say I think it deserves a closer look :).

Reply:
Is this for iGPU test case?

Reply:
Yes we build the kernel module and run it on the AGX Xavier iGPU.

Reply:
I confirm that is not required, at least on dGPU.
For iGPU, should it be required it would be a bug.